tx/textual/valuerenderer/int_test.go

```diff
@@ -64,4 +66,14 @@ func checkNumberTest(t *testing.T, r valuerenderer.ValueRenderer, pv protoreflec
 	require.Equal(t, false, screens[0].Expert)
 
 	require.Equal(t, expected, screens[0].Text)
+	value, err := r.Parse(context.Background(), screens)
+	require.NoError(t, err)
+
+	v, err := math.LegacyNewDecFromStr(pv.String())
+	require.NoError(t, err)
+
+	v1, err := math.LegacyNewDecFromStr(value.String())
+	require.NoError(t, err)
+
+	require.Equal(t, v, v1)
```

Comment:
What's the difference between this and:

```suggestion
	require.True(t, v.Equal(v1))
```